src/main/java/org/jabref/logic/util/strings/HTMLUnicodeConversionMaps.java

```diff
@@ -475,6 +475,7 @@
             {"36", "dollar", "\\$"}, // Dollar
             {"37", "#37", "\\%"}, // Percent (&percnt; is not displayed correctly in the entry preview)
             {"39", "apos", "'"}, // Apostrophe
+            {"39", "apos", "{\\textquotesingle}"}, // Apostrophe                                                          
```

Comment:
Sorry, but I don't think this is going to work this way. This clashes with the already existing mapping for unicode value 39 in the line above and will lead to unpredictable results later.

I think it would be better to add the conversion as part of the "manual additions" to `LATEX_HTML_CONVERSION_MAP` and `LATEX_UNICODE_CONVERSION_MAP` , like for the `relax` term. This should have the same effect, but it avoids the clash for the other maps.

Reply:
So how can it be done? I just do not know the details. I think `{\textquotesingle}` as an alias of `'`.

Reply:
Have a look at lines 897 and 898 in the same class. You can add the symbol in the very same way.